### PR TITLE
Update-developer-site-with-Change-Address-for-MoDe_oT63DyTU

### PR DESCRIPTION
**Pull request by aemson**
- added change address api calls for modify_delivery

### CODE REVIEW

**Comment by bringdev** [comment]
[trello.com/c/oT63DyTU](https://trello.com/c/oT63DyTU)

File: api/modify-delivery/api.raml [hunk 1]
```diff
@@ -70,3 +71,73 @@ documentation:
           application/json:
             example: !include examples/changecodresponse.json
 
+/modifications/{shipmentNumber}/address:
+  displayName: Change address of delivery
```
Comment (nygron) [inline api/modify-delivery/api.raml:75]: Maybe be even more specific: `Change recipient address of delivery.`

File: api/modify-delivery/api.raml [hunk 2]
```diff
@@ -70,3 +71,73 @@ documentation:
...
+
+  uriParameters:
+    shipmentNumber:
+      description: Shipment number to change address for .
```
Comment (nygron) [inline api/modify-delivery/api.raml:79]: remove extra space before .

File: api/modify-delivery/api.raml [hunk 3]
```diff
@@ -70,3 +71,73 @@ documentation:
...
+      type: string
+      required: true
+      example: 707262014721
+
+  post:
+    description: Change address of delivery .
```
Comment (nygron) [inline api/modify-delivery/api.raml:85]: remove extra space before .

File: api/modify-delivery/api.raml [hunk 4]
```diff
@@ -70,3 +71,73 @@ documentation:
...
+      type: string
+      required: true
+      example: 707262014721
+    currentPostalCode:
+      description: Current postal code of recipient address.
+      type: string
+      required: true
+      example: 0021
+    newPostalCode:
+      description: New postal code to where shipment will be re-routed to .
```
Comment (nygron) [inline api/modify-delivery/api.raml:113]: remove space before .

File: api/modify-delivery/api.raml [hunk 5]
```diff
@@ -70,3 +71,73 @@ documentation:
...
+      type: string
+      required: true
+      example: 0484
+
+  get:
+    description: Get price for change address .
```
Comment (nygron) [inline api/modify-delivery/api.raml:119]: remove space before .
This makes me think somehow it's required for the description part. (Or its copy and paste :-D )

**Review by nygron — APPROVED** [review]
Just some trimming of spaces. Awesome job.  Great if you also add it to the change history part.

**Pull request merged**